pkg/control/v1/proto/control_v1_grpc.pb.go

```diff
@@ -20,8 +25,15 @@ import (
 
 // This is a compile-time assertion to ensure that this generated file
 // is compatible with the grpc package it is being compiled against.
-// Requires gRPC-Go v1.32.0 or later.
```

Comment:
Not sure if we document anywhere the minimum version we require for `gRPC-Go` because we jumped a bit forward here

Reply:
that is quite a bit of a jump yes, I didn't find anything that's pinning the grpc-go version. It maybe a good idea for mage commands to attempt to install dependencies rather.